app/src/main/res/layout/activity_home.xml

```diff
@@ -40,9 +40,6 @@
             android:layout_width="match_parent"
```

Comment:
It's better to keep the explicit constraints and set the `layout_width` to `match_constraint` or `0dp` (same thing)

Reply:
Yeah, Right! I will do that. So, do you want me to update the layouts with `constraint layout`?

Reply:
Ohhhh, I forgot that I converted these away from ConstraintLayout! 🤦

Reply:
So, do you want to convert those again to ConstraintLayout? Was there any particular reason to convert away from ConstraintLayout?

Reply:
No particular reason, other than simplicity.

Reply:
Okay.